docs/content/interacting/read-tuple-changes.mdx

```diff
@@ -51,6 +51,14 @@ This section illustrates how to call the Read Changes API to get the list of rel
 3. You have [configured the _authorization model_](../modeling).
 4. You have loaded `FGA_ENVIRONMENT`, `FGA_STORE_ID`, `FGA_CLIENT_ID` and `FGA_CLIENT_SECRET` as environment variables.
 
+</TabItem>
+<TabItem value={SupportedLanguage.PYTHON_SDK} label={languageLabelMap.get(SupportedLanguage.PYTHON_SDK)}>
+
+1. <SdkSetupPrerequisite />
+2. You have [installed the SDK](../getting-started/install-sdk.mdx).
+3. You have [configured the _authorization model_](../modeling).
+4. You have loaded `FGA_ENVIRONMENT`, `FGA_STORE_ID`, `FGA_CLIENT_ID` and `FGA_CLIENT_SECRET` as environment variables.
```

Comment:
I think we missed this in the initial review. This should use the common SDK setup instead of these and also FGA_ENVIRONMENT is not valid for OpenFGA SDKs.

Reply:
updated